Laptops/trackpad-methods/manual.md

```diff
@@ -1,17 +1,48 @@
 # Fixing Trackpads: Manual
 
-* [Finding the ACPI path](#finding-the-acpi-path)
+* [Checking GPI0](#checking-gpio)
 * [Edits to the sample SSDT](#edits-to-the-sample-ssdt)
-* [Compiling the SSDT](#compiling-the-ssdt)
+* [Enabling Trackpad](#enabling-trackpad)
 * [Wrapping up](#wrapping-up)
 
+## Checking GPI0
 
-## Finding the ACPI path
+The first thing which should be checked is the GPI0 device, which is required for VoodooI2C. The best way to check this is working is to use IORegistryExplorer.
 
-Finding the ACPI pathing is quite easy actually, first open your decompiled DSDT you got from [Dumping the DSDT](/Manual/dump.md) and [Decompiling and Compiling](/Manual/compile.md) with either maciASL(if in macOS) or any other text editor if in Windows or Linux(VSCode has an [ACPI extension](https://marketplace.visualstudio.com/items?itemName=Thog.vscode-asl) that can also help).
+![](../../images/Laptops/trackpad-md/gpio-enabled.png)
 
-Next search for `Device (GPI0)`. Should give you a result similar to this:
+Here, we can see that VoodooGPIO is attached to GPI0 so no edits are needed for GPI0. If this is the case for you, you can skip to the [next section](#Enabling-Trackpad).
```

Comment:
Hyperlink is broken, can't have caps:

* `#Enabling-Trackpad` -> `#enabling-trackpad`